src/components/main.js

```diff
@@ -284,8 +244,8 @@ class Main extends React.Component {
     this.unwait();
     this.state.session.disconnect();
     this.setState({ session: null });
-    window.localStorage.removeItem('gridplus_web_wallet_id');
-    window.localStorage.removeItem('gridplus_web_wallet_password');
+    localStorage.removeWalletId()
+    localStorage.removeWalletPassword()
```

Comment:
`removeLoginId` and `removeLoginPassword` (or you could just merge them into `removeLogin`